src/base/settingsstorage.h

```diff
@@ -38,6 +38,18 @@
 
 #include "utils/string.h"
 
+template<typename T>
+struct isQFlags
+    : std::false_type
```

Comment:
This should be in the same line with `struct isQFlags`.
And below too.

Reply:
I would even write it in the following way:
```c++
template <typename T>
struct isQFlags : std::false_type {};
```

Reply:
Since it is a `struct` should it start with uppercase? `IsQFlags`

Reply:
>Since it is a `struct` should it start with uppercase? `IsQFlags`

:+1:
I see no reason to omit this rule in this case.

Reply:
Done.